packages/dataviews/src/field-types/integer.tsx

```diff
@@ -0,0 +1,20 @@
+/**
```

Comment:
Nice file :) 

Should we move the "control" to the field types definition?

Reply:
Yes! And validation, etc. I'll do that in a follow-up.

Reply:
Validation moved to the type definition at https://github.com/WordPress/gutenberg/pull/64164